### ISSUE TITLE
Inaccurate review link scroll

### ISSUE THREAD

**Issue by michaelccata** [18F/e-QIP-prototype#1176]
**What did you do?**

**This is a general issue but here's a specific application.**
Relationships > People who know you well > Review

**What did you see instead?**

Did not insert middle name, eApp did not flag passing onto different block.
<img width="688" alt="screen shot 2018-11-17 at 9 45 03 am" src="https://user-images.githubusercontent.com/4422873/48662257-9354f500-ea4d-11e8-99ff-ba59aeab8083.png">

Clicked review to run validations on section
<img width="437" alt="screen shot 2018-11-17 at 9 39 23 am" src="https://user-images.githubusercontent.com/4422873/48662262-a10a7a80-ea4d-11e8-863d-4240c42f7060.png">

Clicked link and it dropped me down the page to here:
<img width="890" alt="screen shot 2018-11-17 at 9 39 45 am" src="https://user-images.githubusercontent.com/4422873/48662268-b4b5e100-ea4d-11e8-8ff7-6cb96ccacb46.png">

Which is the entry **_after_** Michael Leavers
After scrolling up you then see the actual relevant error

<img width="868" alt="screen shot 2018-11-17 at 9 39 52 am" src="https://user-images.githubusercontent.com/4422873/48662279-de6f0800-ea4d-11e8-811e-3347de8f65e1.png">

**Comment by line47** [comment]
The reason this is happening is because the accordions are closed on the review pages. 
If you expand the accordion then click the link for `Provide this person's full name` it will take you to the right spot in the form. 

I would suggest that on the review pages if an accordion item has errors then load it expanded instead of collapsed.

**Comment by christopherhuii** [comment]
The fix for this was to open up all accordions by default when an applicant is reviewing a section. This will ensure accurate scrolling when an applicant clicks on an error.

However the draw back to this fix is that the form on review pages will get really long and might be difficult to review. I deferred looking for errors because the errors are inconsistent and #1211 should standardize the errors. Once that is implemented, we can decide what we want to do according to the user feedback.

**Comment by michaelccata** [comment]
@christopherhuii Just to clarify: We open all the accordions and we are deferring how to handle the existing phantom or inconsistent errors at a later point once #1211 is finished? Just want to confirm that we are able to open the accordions and scroll appropriately.

**Comment by christopherhuii** [comment]
Currently, the clicking on an error will inaccurately scroll because the accordions are closed.

To fix that, when an applicant is on the "Review" page, I opened up all the accordions, by default. This way, all the fields are displayed. By doing this, when a user clicks on an error, it'll take them directly to the error because the accordion is opened. The user still has to ability to open/close accordions.

Once #1211 is finished and depending on user feedback, we can explore the option of ONLY opening accordions with errors in them, if users find it annoying that all the accordions are open.

**Comment by michaelccata** [comment]
Ahh, ok! Perfect, sounds like a plan.
